velox/functions/prestosql/registration/ArrayFunctionsRegistration.cpp

```diff
@@ -202,11 +202,6 @@ void registerArrayFunctions(const std::string& prefix) {
       Array<Varchar>,
       Array<Varchar>,
       Varchar>({prefix + "array_remove"});
-  registerFunction<
```

Comment:
This is registered twice. Once with `registerArrayRemoveFunctions<Generic<T1>>(prefix);` call and once here. May be we should throw for dual registration.

Reply:
IIRC the registration function has a flag on whether to throw or ignore upon double-registration.